src/parser/query_result.c

```diff
@@ -732,17 +736,46 @@ pt_get_node_title (PARSER_CONTEXT * parser, const PT_NODE * col, const PT_NODE *
 	    {
 	      if (col->info.name.meta_class == PT_META_ATTR)
 		{
-		  name =
-		    pt_append_string (parser, pt_append_string (parser, (char *) col->info.name.resolved, "."), name);
-		  name = pt_append_string (parser, pt_append_string (parser, NULL, "class "), name);
+		  name = pt_append_string (parser, NULL, "class ");
+		  if (parser->custom_print & PT_PRINT_NO_CURRENT_USER_NAME)
```

Comment:
Is there any possibility that PT_PRINT_NO_CURRENT_USER_NAME and PT_PRINT_NO_SPECIFIED_USER_NAME are set at the same time?

Reply:
PT_PRINT_NO_SPECIFIED_USER_NAME implicitly includes PT_PRINT_NO_CURRENT_USER_NAME.
I will not deliberately set both, but since it is not impossible, I will set the priority by checking PT_PRINT_NO_SPECIFIED_USER_NAME before PT_PRINT_NO_CURRENT_USER_NAME.